tasks/page/processGerbers.js

```diff
@@ -29,17 +28,21 @@ if (require.main !== module) {
       file = fs.readFileSync(`${repoRootPath}/kitspace.yml`)
     }
     let info = file == null ? {} : yaml.safeLoad(file)
-    const files = globule
-      .find(`${repoRootPath}/**/*`)
-      .map(p => path.relative(repoRootPath, p))
+    let gerberPath = `${repoRootPath}/**/*`
 
     if (info.multi) {
       for (let project in info.multi) {
         if (project === projectPath) {
           info = info.multi[project]
+          gerberPath = `${repoRootPath}/${projectPath}/**/*`
```

Comment:
We should probably be using `path.join` more to avoid mistakes with extra `/`.

Reply:
Much cleaner then the template literals.